drivers/ieee802154/ieee802154_uart_pipe.c

```diff
@@ -24,9 +24,74 @@
 
 #include "ieee802154_uart_pipe.h"
 
+#define PAN_ID_OFFSET           3 /* Pan Id offset */
+#define DEST_ADDR_OFFSET        5 /* Destination offset address*/
+#define DEST_ADDR_TYPE_OFFSET   1 /* Destination address type */
+
+#define DEST_ADDR_TYPE_MASK     0x0c /* Mask for destination address type */
+
+#define DEST_ADDR_TYPE_SHORT    0x08 /* Short destination address type */
+#define DEST_ADDR_TYPE_EXTENDED 0x0c /* Extended destination address type */
+
+#define PAN_ID_SIZE           2    /* Size of Pan Id */
+#define SHORT_ADDRESS_SIZE    2    /* Size of Short Mac Address */
+#define EXTENDED_ADDRESS_SIZE 8    /* Size of Extended Mac Address */
+
+/* Broadcast Short Address */
+#define BROADCAST_ADDRESS    ((uint8_t [SHORT_ADDRESS_SIZE]) {0xff, 0xff})
+
 /** Singleton device used in uart pipe callback */
 static struct device *upipe_dev;
 
+#if !defined(CONFIG_IEEE802154_UPIPE_PROMISCUOUS)
+
+static bool received_dest_addr_matched(u8_t *rx_buffer)
+{
+	struct upipe_context *upipe = upipe_dev->driver_data;
+	struct ieee802154_context *ctx = net_if_l2_data(upipe->iface);
+
+	/* Check destination PAN Id */
+	if (memcmp(&rx_buffer[PAN_ID_OFFSET],
+			(u8_t *) &ctx->pan_id, PAN_ID_SIZE) != 0 &&
```

Comment:
parameters are always aligned to last '(' + 1 space. 

Apply that comment everywhere